kafka/namespaced/resources/log4j2.yaml

```diff
@@ -0,0 +1,95 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
```

Comment:
I assume this was copied from existing base - do we need to maintain this config?